server/app.js

```diff
@@ -7,13 +7,15 @@ import bodyparser from 'body-parser';
 
 dotenv.config();
 const app = express();
+console.log(app.get('env'));
+
 
 app.use(bodyparser.json());
 app.use(bodyparser.urlencoded({ extended: false }));
 
-app.use('/api/v1/auth/', userRoute);
-app.use('/api/v1/', userRoute);
-app.use('/api/v1/', adminRoute);
+app.use('/api/v2/auth/', userRoute);

```

Comment:
@jo-rdan, please make your routes modular. kindly implement the modularity of your routes in this PR